addons/binding/org.openhab.binding.rotelra1x/README.md

```diff
@@ -0,0 +1,15 @@
+# Rotel amplifier binding
+
+Connects to a Rotel RA11 or RA12 amplifier via a serial interface.
+
+To use it, connect a serial cable between the amplifier and the computer.
+
+Channel summary:
+* Power on/off (soft)
+* Volume. Percentage mapped to level. Max level is user selectable (on RA11 the max is 96, but it may make sense to use 100 then).
+* Mute channel.
+* Source: Selects one of the analogue or digital input channels.
+* Frequency: information only, shows the sampling frequency of the digital input stream.
+* Dimmer: Select the backlight level of the LCD.
+
+All channels are updated in real time if modified from outside of OpenHAB.
```

Comment:
Please format the `channel summary` section in a table with channel IDs etc.

Also, it would be nice if you could include example Things, Items and Sitemap configuration.
See my [README.md](https://github.com/kubawolanin/openhab2-addons/blob/9fb895c346cef84f2523cbc2061231201cc1500f/addons/binding/org.openhab.binding.airquality/README.md) file for reference :-)